packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -636,6 +654,15 @@ export interface CallCompositeStrings {
     stopAllSpotlightMenuLabel: string;
     stopSpotlightMenuLabel: string;
     stopSpotlightOnSelfMenuLabel: string;
+    surveyConfirmButtonLabel: string;
+    surveyIssues: SurveyIssues;
+    SurveyIssuesHeadingStrings: SurveyIssuesHeadingStrings;
+    surveySkipButtonLabel: string;
+    surveyTextboxDefaultText: string;
```

Comment:
align with tagsSurveyTextFieldDefaultText and use `Field` instead of `box`
```suggestion
    surveyTextFieldDefaultText: string;
```

Reply:
This API is actually not being used, I removed it, might be a regression/ something that got missed at some point